index.html

```diff
@@ -0,0 +1,235 @@
+    <button class="intro-button">
+      Start Collaboration
+    </button>
+  </section>
+  <!-- End Introduction Area -->
+
+  <section class="project-banner">
+    <!-- <img src="/assets/images/mobile/projectbanner.svg" width="320px" height="160px" alt=""> -->
+  </section>
+
+  <h1 class="projects-title">Projects</h1>
+
+  <section class="projects">
+    <div class="project-card">
+      <div class="project-image"></div>
+      <div class="project-info">
+        <h3 class="project-title">Project name goes here</h3>
+        <ul class="tech-list">
+          <li class="tech-item">HTML/CSS</li>
+          <li class="tech-item">Ruby on Rails</li>
+          <li class="tech-item">Javascript</li>
+        </ul>
+        <button class="project-detail-btn">
+          See this project
+          <img src="/assets/images/mobile/icon_arrow.svg" alt="project-details-btn" class="details-btn-icon">
+        </button>
+      </div>
+    </div>
+    <div class="project-card">
+      <div class="project-image">
+        <img src="/assets/images/mobile/project2.png" width="100%" height="100%" alt="">
+      </div>
+      <div class="project-info">
+        <h3 class="project-title">Project name goes here</h3>
+        <ul class="tech-list">
+          <li class="tech-item">HTML/CSS</li>
+          <li class="tech-item">Ruby on Rails</li>
+          <li class="tech-item">Javascript</li>
+        </ul>
+        <button class="project-detail-btn">
+          See this project
+          <img src="/assets/images/mobile/icon_arrow.svg" alt="project-details-btn" class="details-btn-icon">
+        </button>
+      </div>
+    </div>
+    <div class="project-card">
+      <div class="project-image">
+        <img src="/assets/images/mobile/project3.png" width="100%" height="100%" alt="">
+      </div>
+      <div class="project-info">
+        <h3 class="project-title">Project name goes here</h3>
+        <ul class="tech-list">
+          <li class="tech-item">HTML/CSS</li>
+          <li class="tech-item">Ruby on Rails</li>
+          <li class="tech-item">Javascript</li>
+        </ul>
+        <button class="project-detail-btn">
+          See this project
+          <img src="/assets/images/mobile/icon_arrow.svg" alt="project-details-btn" class="details-btn-icon">
+        </button>
+      </div>
+    </div>
+    <div class="project-card">
+      <div class="project-image">
+        <img src="/assets/images/mobile/project4.png" width="100%" height="100%" alt="">
+      </div>
+      <div class="project-info">
+        <h3 class="project-title">Project name goes here</h3>
+        <ul class="tech-list">
+          <li class="tech-item">HTML/CSS</li>
+          <li class="tech-item">Ruby on Rails</li>
+          <li class="tech-item">Javascript</li>
+        </ul>
+        <button class="project-detail-btn">
+          See this project
+          <img src="/assets/images/mobile/icon_arrow.svg" alt="project-details-btn" class="details-btn-icon">
+        </button>
```

Comment:
- [ ] The same here image icon in the button is not shown. The path is broken in the remote maybe it works in your VS code/ other editor live extension server locally. It may be tried by adding a dot `.` before starting the path.

Example
```
<img src="./assets/images/mobile/icon_arrow.svg" alt="project-details-btn" class="details-btn-icon">
```
***Your design***
![Screenshot from 2022-12-03 18-37-47](https://user-images.githubusercontent.com/65129228/205450215-5d23c12d-8fdb-4167-b5fa-d19a70c93042.png)